bitrise.yml

```diff
@@ -1234,7 +1245,9 @@ workflows:
             # debug log
             set -x
             # get dependency
-            cd .firefox-ios/test-fixtures && git clone https://github.com/isabelrios/xcresult_extract.git
+            echo "current directory ${PWD}"
+            find . -mindepth 2 -maxdepth 2 -type d
+            cd ./firefox-ios/test-fixtures && git clone https://github.com/isabelrios/xcresult_extract.git
```

Comment:
test-fixtures has not changed, it is going to be in the root directory so that it can be shared by firefox and focus

Reply:
removed the `firefox-ios` in the path.